src/test/java/org/opentripplanner/standalone/config/ConfigLoaderTest.java

```diff
@@ -88,6 +89,38 @@ public void parseJsonString() {
         assertEquals("value", node.path("key").asText());
     }
 
+    @Test
+    public void testResolveEnvironmentVariables() {
+        // Given: a environment variable (with a alphanumeric name less then 20 characters)
```

Comment:
We decided to expand the explanation of this test because it's using whatever environment variables happen to be floating around, which might not be intuitive to the reader. And this could potentially fail on some system, so we want the reader to immediately understand how it works.

Reply:
I did add more comments, as well as a unit test on the EnvironmentVariableReplacer class.